src/decorator/options/ValueTransformer.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Interface for objects that deal with (un)marshalling data.
+ */
+export interface ValueTransformer<P, Q> {
```

Comment:
Also I don't think we need `<P, Q>`. They arent really used anywhere and does not really provide type safety anywhere

Reply:
@pleerock I think they're useful. If you accidentally did something like

```javascript
transform: {
  from(v: string) {
    return v.split(",");
  },
  to(v: string[]) {
    return v.concat(","); // obvious mistake here, should use .join
  }
}
```

Then the template parameters would catch that this is not a valid `ValueTransformer`.